### PR TITLE
Adding webP compression to image assets and animation (wind) to the robe displayed

### PR DESCRIPTION
**Pull request by arckoor**

### CODE REVIEW

**Comment by Coldknife2** [comment]
45ecf578ac8851d94a4e7789a0f5576f895fef37 could induce confusion for any Vue dev joining the project, a quick mention in README / Contributing.md could be a great thing. 
(So that they know about possible naming conflicts, and the way imports works in this project)
That's the kind of thing that should be documented

File: scripts/webpCompression.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+			process.exit(9);
+		}
+		return temp;
+	}
+	return def;
+}
+
+function parsePresets(set) {
+	const presets = [
+		"cwebp -m 6 -q 100 -mt -v",
```
Comment (Coldknife2) [inline scripts/webpCompression.js:53]: "cwebp" will this command run on any machine ? Haven't seen a mention in the package.json

**Comment by Coldknife2** [comment]
Nice work for this PR again. I would change the title however, normally you would be able to make sense from all commits you're showing in your PR.
If not, it may be an indicator of a PR being too big or a "stuff-all-you-can" PR

Proposing: "Adding webp compression to assets and animation (wind) to the robe displayed"
It's up to you to either put more focus on a fix that is potentially breaking change, or leave them in silence
Commit naming convention != PR naming convention

File: scripts/webpCompression.js (hunk 1 above)
Comment (arckoor) [reply scripts/webpCompression.js:53]: https://github.com/Coldknife2/JourneySaveParserHtml/pull/18/commits/f542d8888b5e216c327eb3eb076b82e036172706 resolves that by checking if the needed binaries are available.

**Pull request merged**